src/components/Tables/components/TableActionMenu.tsx

```diff
@@ -0,0 +1,63 @@
+import { DropdownMenu } from '@radix-ui/themes'
+import { DotsVerticalIcon } from '@radix-ui/react-icons'
+import { useState } from 'react'
+import { ConfirmDelete } from './ConfirmDelete'
+
+type Props = {
+    isFeatured: boolean
+    onToggleFeatured: () => void
+    onEdit: () => void
+    onDelete: () => void
+}
+
+export function TableActionMenu({
+    isFeatured,
+    onToggleFeatured,
+    onEdit,
+    onDelete
+}: Props) {
+    const [deleteDialogOpen, setDeleteDialogOpen] = useState(false)
+
+    return (
+        <>
+            <DropdownMenu.Root>
+                <DropdownMenu.Trigger>
+                    <DotsVerticalIcon className="cursor-pointer" />
+                </DropdownMenu.Trigger>
+                <DropdownMenu.Content size="2">
+                    <DropdownMenu.Item
+                        onClick={onEdit}
+                    >
+                        Edit
+                    </DropdownMenu.Item>
+                    <DropdownMenu.Separator />
+                    <DropdownMenu.Item
+                        color={isFeatured ? undefined : 'green'}
```

Comment:
Just wondering why you removed cursor pointer from the items?

Reply:
When I changed it to tailwind class it didn't work on those elements. I decided that the color change onHover was enough feedback for the user, but we could change it back.

Reply:
Yes, please figure out why cursor pointer isn't working. You can always next a `Box` inside the item and add the class to that.